### DISCUSSION TITLE
I have installed pimcore on rhel...while uploading csv file in asset It is showing the below error

### DISCUSSION THREAD

**Discussion by gunnu1303** [pimcore/pimcore#14386] [Q&A / Support]
I have increased the upload_max_size and post_max_size to 500M in etc/php.ini but still its showing the same error.
[
![Screenshot 2023-02-19 001605](https://user-images.githubusercontent.com/123451461/219882972-eebe92a9-2dbf-458d-9d01-037ad2777964.png)
](url)

**Comment by blankse** [comment]
Did you reload the backend after that?

**Reply by gunnu1303** [reply]
Yess...I always restart apache after doing changes in php.ini files.

**Comment by blankse** [comment]
The php setting is named upload_max_filesize, not upload_max_size.

**Reply by gunnu1303** [reply]
Yess sorry my mistake. I accidentally wrote size instead of filesize. But i did changes in filesize. Still it's showing this error
